static/app/views/performance/queues/queuesTable.tsx

```diff
@@ -73,17 +74,34 @@ const COLUMN_ORDER: Column[] = [
   },
 ];
 
+const SORTABLE_FIELDS = [
+  'messaging.destination.name',
+  'time_spent_percentage()',
+] as const;
+
+type ValidSort = Sort & {
+  field: (typeof SORTABLE_FIELDS)[number];
+};
+
+export function isAValidSort(sort: Sort): sort is ValidSort {
+  return (SORTABLE_FIELDS as unknown as string[]).includes(sort.field);
```

Comment:
Coercing it to unknown seems a little weird to me. Did a quick search and it seems like we can do this

```suggestion
  return (SORTABLE_FIELDS as ReadonlyArray<string>).includes(sort.field);
```

which gives us a touch more type safety by checking if SORTABLE_FIELDS is compatible with the assertion

Reply:
Thanks! Updated!